lib/datadog/core/vendor/multipart-post/multipart.rb

```diff
@@ -6,8 +6,10 @@
 #++
 
 module Datadog
-  module Vendor
-    module Multipart
+  module Core
+    module Vendor
+      module Multipart
```

Comment:
I'm curious: is any product other than profiling using multipart-post? If not, it may be worth throwing this inside the profiling folder, so it can be deleted [once we move to libddprof](https://github.com/DataDog/libddprof).

Reply:
It isn't at the moment, but it's (1) an external library that was imported, and (2) it's will likely have immediate application in the near future for other transport activity.

I think its different than other components like `Event` which was an abstraction we created for the purposes of tracing; instead, this is something that implements an external RFC (multipart HTTP posts) that has broad applications outside profiling.

As such, I'd like to keep this here.